psalmist_frontend/src/components/pages/NewSong.js

```diff
@@ -72,24 +76,27 @@ const NewSong = ({ song: { current, song }, addSong, getSong }) => {
             <button>Home</button>
           </a>
           <Editor
+            apiKey="1v1rtd8b64vx0ekbzfubrdydw3skdnd67tuq9924fu8h8tde"
```

Comment:
❌ This API key should be stored in the `.env` file. It is a bad practice to have secrets committed to version control